src/device-monitoring/routes.py

```diff
@@ -0,0 +1,10 @@
+BASE_URL = '/api/v1/monitor'
+
+HEALTH = '/health'
```

Comment:
Let's use `f'{BASE_URL}/health'` too - otherwise we won't be able to access this route once its deployed